docs/sftt/sftt-api-spec.md

```diff
@@ -344,11 +399,11 @@ No request body.
 
 ##### `200 OK`
 
-!!! missing "Unknown"
+Applied successfully.
 
 ##### `400 BAD REQUEST`
 
-!!! missing "Should be an error here if the team id is invalid or if the user is already on the team"
+If the team id is invalid or if the user is already on the team"
```

Comment:
Is a user allowed to apply / get invited to a team if they were already on the team and left? If so then we should specify _somewhere_ that "being on the team" means that the user has a team_role that is not equal to NONE or PENDING